### PR TITLE
Add "Getting Security-Hint programmatically" section

### PR DESCRIPTION
**Pull request by OneLineTwoBugs**
Hi,

I used your guide to read variables from my Logo - thanks a lot for making your findings public, they were very helpful to me. I spent some time yesterday to reverse engineer the LOGOs login process and added my findings to your document. Hope you find this worthy of inclusion to your work.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,72 @@ Trying to make sense of the Siemens LOGO sort-of built-in REST API. Maybe, somed
...
+curl_setopt($ch, CURLOPT_POST, true);
+curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+curl_setopt($ch, CURLOPT_POSTFIELDS, "UAMCHAL:3,4,$a1,$a2,$b1,$b2");
+
+$ret = curl_exec($ch); //700,ABCDEF1234567890ABCDEF1234567890,1234567890
+$ret = explode(",", $ret);
+if (count($ret) == 3 && $ret[0] == "700") {
+  $pwToken = substr($pw . '+' . $ret[2], 0, 32);
+  $iPWToken = gmp_xor(crc32($pwToken), $ret[2]);
+  $iServerChallenge = gmp_xor(gmp_xor(gmp_xor(gmp_xor($a1, $a2), $b1), $b2), $challenge);
```
Comment (fafr84) [inline README.md:75]: $iServerChallenge = gmp_xor(gmp_xor(gmp_xor(gmp_xor($a1, $a2), $b1), $b2), $ret[2]);

**Comment by fafr84** [comment]
Hi, it's working only inside your PHP script.
But when taking the generated key and do a request outside the script it won't work.

**Comment by OneLineTwoBugs** [comment]
Interesting. I use the token from within the same script, hence I've never noticed this. I'll investigate further next week.

File: README.md (hunk 1 above)
Comment (OneLineTwoBugs) [reply README.md:75]: sorry, this fix is of course correct. It happened when I've tidied up my script for the pull request. I'll fix this

**Comment by jankeymeulen** [comment]
Thanks for figuring this out!

**Pull request merged**